infrastructure/Taskfile.yml

```diff
@@ -1046,10 +1046,18 @@ tasks:
         Gets the latest deployment for each production environment and prints its status if it is *not* complete.
       cmds:
         - |
-          lagoon list projects --output-json \
-            | jq -r '.data[].projectname'  \
-            | while read -r projectname; do echo "$projectname: $(lagoon list deployments -e main -p $projectname --output-json | jq '.data[0].status')"; done \
-            | grep -v "complete"
+          lagoon raw --raw "query allProjects {
```

Comment:
I'm getting an error when running this. It says that it doesn't know `raw`. It seems that `dplsh`'s cli is outdated? What version are you running?

Reply:
You are right.

I am runing `lagoon v0.21.3 (go-1.21)` locally but DPLSH has `lagoon v0.20.2 (go-1.16)`